cmd/jaeger/internal/integration/span_reader.go

```diff
@@ -82,7 +82,7 @@ func (r *spanReader) GetTrace(ctx context.Context, traceID model.TraceID) (*mode
 		for i := range received.Spans {
 			spans = append(spans, &received.Spans[i])
 		}
-		r.logger.Info(fmt.Sprintf("GetTrace received %d spans (total %d)", len(received.Spans), len(spans)))
+		// r.logger.Info(fmt.Sprintf("GetTrace received %d spans (total %d)", len(received.Spans), len(spans)))
```

Comment:
@james-ryans are you sure we need this log? It's very noisy, I don't think it adds much value other than a form of progress reporting (which I would do with much lighter approach, maybe logging every 1000 spans)